data/remnant ships.txt

```diff
@@ -559,3 +695,57 @@ ship "Starling" "Starling (Thrasher)"
 	gun
 
 
+
+ship "Tern"
+	sprite "ship/tern"
+	thumbnail "thumbnail/tern"
+	attributes
+		category "Drone"
+		licenses
+			Remnant
+		"cost" 2530000
+		"shields" 500
+		"hull" 2500
+		"cargo space" 5
+		"fuel capacity" 50
+		"energy capacity" 500
+		"ramscoop" 0.5
+		"automaton" 1
+		"mass" 32
+		"drag" .8
+		"heat dissipation" .9
+		"gaslining" 1
+		"outfit space" 75
+		"shield generation" 0.6
+		"shield energy" 0.4
+		"hull repair rate" 1.2
+		"hull energy" 0.9
+		"atmosphere scan" 100
+		"outfit scan power" 12
+		"outfit scan speed" 1
+		"tactical scan power" 26
+		"asteroid scan power" 30
+		"weapon capacity" 16
+		"engine capacity" 26
```

Comment:
Weapon capacity and engine capacity positions out of order.

Reply:
Fixed.

Reply:
The order was fine, they're just in the completely wrong position (should be up below outfit space).

Reply:
OK. Moved.